docs/userguides/accounts.md

```diff
@@ -66,6 +66,16 @@ ape accounts import <ALIAS>
 It will prompt you for the private key.
 If you need help exporting your private key from Metamask, see [this guide](https://metamask.zendesk.com/hc/en-us/articles/360015289632-How-to-export-an-account-s-private-key).
 
+You can also import accounts from mnemonic seed by using the --mnemonic flag:
+
+```bash
+ape accounts import --mnemonic <ALIAS>
+```
+
+It will prompt you for the [mnemonic seed](https://metamask.zendesk.com/hc/en-us/articles/360015290032-How-to-reveal-your-Secret-Recovery-Phrase). 
+After this it will prompt you for the [HDPath](https://help.myetherwallet.com/en/articles/5867305-hd-wallets-and-derivation-paths) to the account you'd like to use with this seed phrase.
```

Comment:
I think this would align with the trezor/ledger workflow as well.

Reply:
The `Input HD Path to use [m/44'/60'/0'/0/0]:` prompt now only appears if the `--hdpath` flag is used. Documentation and tests have also been updated for this change.

Reply:
I am having trouble using `--hdpath` when not a flag, since `m/44'/61'/0'/0/0` needs to be wrapped in quotes like this `"m/44'/61'/0'/0/0"` or else it causes trouble with bash cli

Reply:
`needs to be wrapped in quotes like this "m/44'/61'/0'/0/0" or else it causes trouble with bash cli`
after discussion, this is fine and expected